gcs/src/it/scala/org/broadinstitute/monster/storage/gcs/GcsApiIntegrationSpec.scala

```diff
@@ -122,7 +126,7 @@ class GcsApiIntegrationSpec
   it should "read entire objects as a stream" in {
     writeTestFile.use { blob =>
       withClient(api => buildString(api.readObject(blob.getBucket, blob.getName)))
-    }.unsafeRunSync() shouldBe bodyText
+    }.unsafeRunSync() shouldBe buildString(body).unsafeRunSync()
```

Comment:
It'd be better to only `unsafeRunSync()` once per test. In this case, the setup would be like:
```scala
val checks = for {
  chunk <- bodyText(someNumber).compile.toChunk
  writtenBytes <- writeTestFile(chunk).use { blob =>
    withClient(...)
  }
  _ <- writtenBytes shouldBe chunk
} yield ()
checks.unsafeRunSync()
```